app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    try:
+        mv_command, source_file, dstn_path = command.split()
```

Comment:
`dstn_path` bad variable name 🥲

Reply:
Thank you for highlighting this!
Changed `dstn_path` to `destination`. It will be better?